autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py

```diff
@@ -253,10 +253,12 @@ def get_hyperparameter_search_space(
         dataset_properties: Optional[Dict] = None,
         batch_size: HyperparameterSearchSpace = HyperparameterSearchSpace(hyperparameter="batch_size",
                                                                           value_range=(32, 320),
-                                                                          default_value=64)
+                                                                          default_value=64,
+                                                                          log=True)
```

Comment:
Does it make sense to do `log=True` search of the batch size?

Reply:
I would say it makes sense since we might be more biased towards lower batch sizes. Additionally, it was in the AutoPyTorch paper search space: https://arxiv.org/pdf/2006.13799.pdf